src/udev/udev-builtin-net_id.c

```diff
@@ -1082,7 +1093,11 @@ static int get_link_info(sd_device *dev, LinkInfo *info) {
         if (r < 0 && r != -ENOENT)
                 return r;
 
-        (void) sd_device_get_sysattr_value(dev, "phys_port_name", &info->phys_port_name);
+        r = sd_device_get_sysattr_value(dev, "phys_port_name", &info->phys_port_name);
+        if (r >= 0 && sscanf(info->phys_port_name, "pf%uvf%u", &pf, &vf) == 2)
+                info->vf_repr_num = vf;
```

Comment:
this is a bit confused over the type. it parses the thing as unsigned into an unsigned, and then writes it to this signed integer. so what is it? signed or unsigend?

please stick to one type across the codebase for the same concept.

Reply:
OK, I wanted to use -1 as 'this is not representor' and wouldn't like to allow to parse strings like 'pf0vf-20'... I think we can use `int` as the kernel never pass string like this.